client/src/components/UI/ProfilePage.jsx

```diff
@@ -7,16 +7,15 @@ import "../../assets/css/hidden.css";
 import UserDashboard from "../miniComponents/UserDashboard";
 import InstructorDashboard from "../miniComponents/InstructorDashboard";
 
-function IsInstructor({ IsInstructor }) {
-  if (IsInstructor) {
-    console.log("Is instructor");
+function IsInstructor() {
+  const role = JSON.parse(sessionStorage.getItem("user")).role[1];
```

Comment:
Needs to be changed to `.role` instead of `.role[1]` to reflect the current logic